org/party.go

```diff
@@ -122,14 +124,16 @@ type Website struct {
 // The definition found here is based on the details required for spain.
 // If your country requires additional fields, please let us know.
 type Registration struct {
-	UUID    *uuid.UUID `json:"uuid,omitempty" jsonschema:"title=UUID"`
-	Office  string     `json:"office,omitempty" jsonschema:"title=Office"`
-	Book    string     `json:"book,omitempty" jsonschema:"title=Book"`
-	Volume  string     `json:"volume,omitempty" jsonschema:"title=Volume"`
-	Sheet   string     `json:"sheet,omitempty" jsonschema:"title=Sheet"`
-	Section string     `json:"section,omitempty" jsonschema:"title=Section"`
-	Page    string     `json:"page,omitempty" jsonschema:"title=Page"`
-	Entry   string     `json:"entry,omitempty" jsonschema:"title=Entry"`
+	UUID     *uuid.UUID    `json:"uuid,omitempty" jsonschema:"title=UUID"`
+	Capital  *num.Amount   `json:"social_capital,omitempty" jsonschema:"title=Capital"`
```

Comment:
@samlown How about a `Capital` and `Currency` combination? I'm wary about leaving the `Capital` field as a simple string... FatturaPA for example require a numeric format and it would be good if we can validate it as such. 

Have you considered something like a `Money` type that always includes a currency and an amount by the way?

Reply:
Yeah, I think that's okay. Saying "what's your registration capital" or "what's your registration currency", sounds reasonable.